packages/components/tabs/src/tabs.tsx

```diff
@@ -143,7 +145,11 @@ export default defineComponent({
       emit('edit', undefined, 'add')
       emit('tabAdd')
     }
-
+    const TabNavRenderer: FunctionalComponent<{
+      container: VNode
+    }> = (props, { slots }) => {
+      return <Teleport to={props.container.el}>{slots.default?.()}</Teleport>
```

Comment:
Sorry for the late reply.
Base on what you pointed out, I rethought the solution of this problem , and I‘m going to use flex layout to control the rendering order of `header` and `content` instead of `Teleport`.  But it could affect some users, I'm not sure that it's worth.